app/templates/direct-award/save-search.html

```diff
@@ -23,10 +23,32 @@
 
 <div class="single-question-page save-search-page">
   <div class="grid-row">
+    <div class="column-one-whole">
+        {% if form.name.errors or save_search_selection_error %}
+        <div class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-2" tabindex="-1">
+
+          <div class="dmspeak">
+            <h2 class="heading-xmedium">There was a problem saving your search</h2>
+          </div>
+
+          <ul class="error-summary-list">
+              {% if save_search_selection_error %}
+                <li><a>{{ save_search_selection_error }}</a></li>
+              {% endif %}
+              {% if form.name.errors and not save_search_selection_error %}
+                {% for error in form.name.errors %}<li><a>{{ error }}</a></li>{% endfor %}
+              {% endif %}
+          </ul>
+    
+        </div>
+        {% endif %}
+    </div>
+  </div>
+  <div class="grid-row">
     <div class="column-two-thirds">
       <header class="page-heading-smaller page-heading-without-breadcrumb">
         <h1>
-          Save your search
+            Choose where to save your search
```

Comment:
Not that it matters particularly but not sure why this has extra indentation - in templates we generally use 2 spaces (for whatever reason).